xml/security_ldap_modules.xml

```diff
@@ -25,7 +29,9 @@ ACL preoperation
 [...]</screen>
   <para>
     The following command enables the MemberOf plugin referenced in
-    <xref linkend="sec-security-ldap-server-sssd"/>:
+    <xref linkend="sec-security-ldap-server-sssd"/>. MemberOf simplifies
+    user searches by returning the user and any groups the user
+    belongs to, with a single command:
```

Comment:
You could expand on this further, stating that memberOf allows a client to do a single lookup to retrieve the groups of a user, rather than needing to do multiple lookups :)